resources/scripts/ttx_diff.py

```diff
@@ -387,10 +387,14 @@ def normalize_glyf_contours(ttx):
         normalized = sorted(contours, key=to_xml_string)
         if normalized == contours:
             continue
+        # normalized contours should be inserted before any other TTGlyph's
+        # subelements (e.g. instructions)
         for contour in contours:
             glyph.remove(contour)
-        for contour in normalized:
-            glyph.append(contour)
+        non_contours = list(glyph)
+        for el in non_contours:
+            glyph.remove(el)
```

Comment:
Since you are now removing everything I would imagine you can remove both remove loops and just call something to drop all child elements before you add everything back?

Reply:
we need a list of elements that are not `<contours>` to be placed at the end after the normalized ones, we could alternatively drop all child elements and collect the ones that aren't contours in one for loop.. but it wouldn't change much. feel free to edit